### PR TITLE
Update additional Equipment effect descriptions and icons

### PR DESCRIPTION
**Pull request by ammalagonc**
- Also add icons to some equipment items
- Fix some lint

### CODE REVIEW

File: packs/equipment-effects/effect-boulderhead-bock.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Boulderhead Bock",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Boulderhead Bock].</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Boulderhead Bock].</p>\n<p>You gain a +1 item bonus to saving throws against effects that would make you stunned or stupefied.</p>\n<p>You take a -2 item penalty to Perception for 1 hour.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-boulderhead-bock.json:7]: Consider updating the predicate to `"inflicts:stunned"` and `"inflicts:stupefied"`.

File: packs/equipment-effects/effect-bewitching-bloom-magnolia.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
     "_id": "qLl1jwybXY6EbOoI",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/magic/symbols/arrowhead-green.webp",
     "name": "Effect: Bewitching Bloom (Magnolia)",
     "system": {
         "description": {
-            "value": "<p>Your ally seems more charming, though no one can pinpoint why. They gain a +1 status bonus to Diplomacy checks for 10 minutes. Once on their next turn, the ally can attempt to draw a creature's attention. To do so, the ally must spend 1 action (which has the emotion, mental, and visual traits) to select a creature the ally can see and make eye contact in a way the target can see. The target must succeed at a @Check[will|dc:20] save or be @UUID[Compendium.pf2e.conditionitems.Item.Fascinated] with your ally until the end of that ally's next turn. If the save succeeds or the effect ends, the target is temporarily immune to being fascinated by any magnolia bewitching bloom for 24 hours.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Bewitching Bloom (Magnolia)]</p>\n<p>You gain a +1 status bonus to Diplomacy checks for 10 minutes.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-bewitching-bloom-magnolia.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Bewitching Bloom (Magnolia)]</p>\n<p>You gain a +1 status bonus to Diplomacy checks.</p>"
```

File: packs/equipment-effects/effect-bewitching-bloom-lotus.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
     "_id": "ri5qxyVViva60ilN",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/magic/symbols/arrowhead-green.webp",
     "name": "Effect: Bewitching Bloom (Lotus)",
     "system": {
         "description": {
-            "value": "<p>A glow envelops your ally as a sense of enlightened peace ripples out from within. The ally gains a +2 status bonus to Will saves against mental effects for 1 minute. This bonus increases to +3 against emotion effects.</p>"
+            "value": "<p>Grant by @UUID[Compendium.pf2e.equipment-srd.Item.Bewitching Bloom (Lotus)]</p>\n<p>You gain a +2 status bonus to Will saves against mental effects for 1 minute. This bonus increases to +3 against emotion effects.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-bewitching-bloom-lotus.json:7]:
```suggestion
            "value": "<p>Grant by @UUID[Compendium.pf2e.equipment-srd.Item.Bewitching Bloom (Lotus)]</p>\n<p>You gain a +2 status bonus to Will saves against mental effects. This bonus increases to +3 against emotion effects.</p>"
```

File: packs/equipment-effects/effect-beastmasters-sigil-armor-greater.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "gZOED4T3o6giterN",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/reptiles/lizard-iguana-green.webp",
```
Comment (simonward) [inline packs/equipment-effects/effect-beastmasters-sigil-armor-greater.json:3]: Did you intend for the greater, major, and normal to have different icons?

File: packs/equipment-effects/effect-beastmasters-sigil-melee-weapon-greater.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "gU3uZE2ihLnpQN0b",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/mammals/wolf-shadow-black.webp",
```
Comment (simonward) [inline packs/equipment-effects/effect-beastmasters-sigil-melee-weapon-greater.json:3]: Did you intend for the greater, major, and normal to have different icons?

File: packs/equipment-effects/effect-beastmasters-sigil-ranged-weapon-greater.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
     "_id": "TU67AK08CUsP7pl4",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/creatures/mammals/bat-giant-tattered-purple.webp",
```
Comment (simonward) [inline packs/equipment-effects/effect-beastmasters-sigil-ranged-weapon-greater.json:3]: Did you intend for the greater, major, and normal to have different icons?

Comment (ammalagonc) [reply packs/equipment-effects/effect-beastmasters-sigil-ranged-weapon-greater.json:3]: Different animals per the description, so I was going for that.

File: packs/equipment-effects/effect-beastmasters-sigil-melee-weapon-greater.json (hunk 5 above)
Comment (ammalagonc) [reply packs/equipment-effects/effect-beastmasters-sigil-melee-weapon-greater.json:3]: Different animals per the description, so I was going for that.

File: packs/equipment-effects/effect-beastmasters-sigil-armor-greater.json (hunk 4 above)
Comment (ammalagonc) [reply packs/equipment-effects/effect-beastmasters-sigil-armor-greater.json:3]: Different animals per the description, so I was going for that.

File: packs/equipment-effects/effect-bewitching-bloom-lotus.json [hunk 7]
```diff
@@ -1,10 +1,10 @@
 {
     "_id": "ri5qxyVViva60ilN",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/magic/symbols/arrowhead-green.webp",
     "name": "Effect: Bewitching Bloom (Lotus)",
     "system": {
         "description": {
-            "value": "<p>A glow envelops your ally as a sense of enlightened peace ripples out from within. The ally gains a +2 status bonus to Will saves against mental effects for 1 minute. This bonus increases to +3 against emotion effects.</p>"
+            "value": "<p>Grant by @UUID[Compendium.pf2e.equipment-srd.Item.Bewitching Bloom (Lotus)]</p>\n<p>You gain a +2 status bonus to Will saves against mental effects. This bonus increases to +3 against emotion effects.</p>"
```
Comment (TikaelSol) [inline packs/equipment-effects/effect-bewitching-bloom-lotus.json:7]: Make the second RE an AdjustModifier with `item:trait` as well

File: static/lang/re-en.json [hunk 8]
```diff
@@ -920,6 +920,9 @@
             "Falrok": {
                 "LazuriteTerrainSwitch": "In lazurite infused terrain"
             },
+            "FayatiAlummur": {
+                "GuildmastersLead": "Guildmaster's Lead"
+            },
             "FiveColorOrchidMantis": {
```
Comment (TikaelSol) [inline static/lang/re-en.json:926]:
```suggestion
            },
            "FiveColorOrchidMantis": {
```

**Pull request merged**